Source/Helpers/YPPermissionCheckable.swift

```diff
@@ -45,6 +45,8 @@ extension YPPermissionCheckable where Self: UIViewController {
                     block(granted)
                 }
             })
+        @unknown default:
```

Comment:
Remove `@unknown default` from the MR and merge from last commit from master branch.